Packs/AtlassianConfluenceCloud/README.md

```diff
@@ -0,0 +1,4240 @@
+Atlassian Confluence Cloud allows users to interact with confluence entities like content, space, users and groups. Users can also manage the space permissions.
+This integration was integrated and tested with version 1000.0.0-847bdcbfcd00 of Atlassian Confluence Cloud.
+
+## Configure Atlassian Confluence Cloud on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
```

Comment:
The integration README and the pack README shouldn't be the same.
The pack README should contain general information about the pack and its usage.
See the `nutanix` pack as an example.